docs/api/modules/State-Management.md

```diff
@@ -0,0 +1,153 @@
+
+## Higher Order Components
+
+[Holocron Modules](../API.md#modules) use [Higher Order Components (HOC)](https://reactjs.org/docs/higher-order-components.html) to add behaviors regarding 1) when a Module loads, 2) connecting a Module with its [Reducer(s)](https://redux.js.org/basics/reducers/) to a Redux Store (similar to [Redux `connect`](https://react-redux.js.org/api/connect)) and 3) adding runtime validations for a Module.
+
+**Contents**
+* [`holocronModule`](#holocronmodule)
+
+### `holocronModule`
+
+Please see [`holocronModule`](https://github.com/americanexpress/holocron/blob/master/packages/holocron/API.md#holocronmodule) in the Holocron API.
+
+## Shared Ducks
+
+One App uses a default set of shared Redux [Ducks](https://github.com/erikras/ducks-modular-redux) (e.g. a design pattern for logical groupings of Reducers, Actions, and Selectors that depend on each other) to manage the state of core features (e.g. Loading Language Packs or Holocron Modules) on the Server and Browser environments.
+
+The following is an overview of the shape One App provides:
+
+```
+{
+  config,
+  errorReporting,
+  modules,
+  error,
+  browser,
+  rendering,
+  redirection,
+  holocron,
+  intl
+}
+```
+
+The following API definitions describe the Ducks responsible for the state shape described above.
+
+**Contents**
+* [`config`](#config-duck)
+* [`errorReporting`](#errorreporting-duck)
+* [`error`](#error-duck)
+* [`browser`](#browser-duck)
+* [`rendering`](#rendering-duck)
+* [`redirection`](#redirection-duck)
+* [`holocron`](#holocron-duck)
+* [`intl`](#intl-duck)
+
+### `config` Duck
+The `config` Duck lists a subset of the environment variables set on the Server.
+
+**Contents:**
+* [State Shape](#state-shape)
+* Action Creators
+  * [`setConfig`](#setconfig)
+
+#### State Shape
+
+```js
+const state = new Map({
+  config: new Map({
+    // URL where the Browser sends client side errors to.
+    reportingUrl: String,
+    // URL where the One App static assets are located.
+    cdnUrl: String,
+    // The exact filename for the locale file used in the Browser.
+    localeFilename: String,
+    // Name of the Holocron Module that serves as the entry point to your application.
+    rootModuleName: String,
+    // ... Settings from provideStateConfig key values will land here.
+    [provideStateConfigSettingName]: String,
+  }),
+  // ... Rest of Redux State
+});
+```
+
+**📘 More Information**
+* Adding values to `config` state with [`provideStateConfig`](./App-Configuration.md#providestateconfig) from [App Configuration API](./App-Configuration.md).
+* Learn more about [Environment Variables](#):
+  * [`ONE_CLIENT_REPORTING_URL`](#),
+  * [`ONE_CLIENT_CDN_URL`](#),
+  * [`ONE_CLIENT_LOCALE_FILENAME`](#),
+  * [`ONE_CLIENT_ROOT_MODULE_NAME`](#)
+
+#### Action Creators
+
+##### `setConfig`
+
+> ⚠️ For Internal Use
+
+**Shape**
```

Comment:
shape? I'd expect this to be `**Usage**`, followed by `**Arguments**` after the code block

Reply:
We use `**Shape**` elsewhere: https://github.com/americanexpress/one-app/pull/19/files/1f14dc8aabda592857072b3822bee43cb70de4b9#diff-2919092b5b3a7dfbd26a8e11b7650597R17-R20

Ill add `**Arguments**` header.